api/app/fireweather_bot/hourly_actuals.py

```diff
@@ -44,7 +46,44 @@ def _fix_datetime(source_date):
     return python_date
 
 
-class HourlyActualsBot(BaseBot):
+def parse_hourly_actual(station_code: int, hourly_reading: WeatherReading):
+    """ Maps WeatherReading to HourlyActual """
+    temp_valid = hourly_reading.temperature is not None
+    rh_valid = hourly_reading.relative_humidity is not None and validate_metric(
```

Comment:
Is there actually data that is coming through with crazy values?

Reply:
@conbrad I've been thinking - and I think it's worth hitting out the WF1 team and confirming with them how they handle invalid values. I think in the past they've used some strange approaches, like having one invalid reading mean that other readings are also invalid. So it may be the case, that if one of the readings are wrong, we need to disregard all other readings.